pallets/balances/src/lib.rs

```diff
@@ -319,6 +319,13 @@ pub mod pallet {
         /// Final argument indicates the destination balance type.
         /// \[from, to, balance, destination_status]
         ReserveRepatriated(T::AccountId, T::AccountId, Balance, Status),
+        /// Transfer with memo succeeded.
+        TransferWithMemo {
```

Comment:
This event name doesn't match the one added to the Substrate balance pallet:
https://github.com/PolymeshAssociation/polkadot-sdk/blob/136eaf3cb3eeb76592cbcbc85bf3cffcfd14b0b0/substrate/frame/balances/src/lib.rs#L406
```suggestion
        /// Transfer with memo succeeded.
        TransferMemo {
```

Reply:
Or maybe we want to rename the event in the Substrate repo.

Reply:
Yeah. My plan was to rename the event in the Substrate repo

Reply:
It looks like you've inserted TransferWithMemo in the middle of the Event enum. That changes the variant indices of subsequent events. Are we okay with that? Often we append new event variants at the end of the enum to avoid shifting existing variant indices.  Thoughts?

Reply:
Looks like the new event is at the end.